### PR TITLE
Update some icons in the toolbar (bug 1739965)

### PR DESCRIPTION
**Pull request by calixteman**
Firefox UI/UX team provided us some new icons to use for the visible toolbar and for the ink cursor.
They didn't add one for the "open file" button so I just rotated the arrow myself from the "download" icon, I asked to have another one so it should be temporary.

### CODE REVIEW

**Review by Snuffleupagus — COMMENTED** [review]
This only updates a small number of the existing icons, which makes the toolbar look inconsistent (it looks slightly jarring with the styles being so different).

Is the intention to actually update *all* of the icons "soon", e.g. in the coming days or maybe week?
In that case, I think we can just hold off on landing this until all new icons are available.
Otherwise, it's not clear to me that we'd want to change only some of the icons.

---

Slightly unrelated, and definitely not your "fault", but on a *normal* screen (i.e. non-HiDPI) these icons feel slightly "blurrier" than the old ones when using the viewer.

**Comment by calixteman** [comment]
> This only updates a small number of the existing icons, which makes the toolbar look inconsistent (it looks slightly jarring with the styles being so different).
> 
> Is the intention to actually update _all_ of the icons "soon", e.g. in the coming days or maybe week? In that case, I think we can just hold off on landing this until all new icons are available. Otherwise, it's not clear to me that we'd want to change only some of the icons.
> 
> Slightly unrelated, and definitely not your "fault", but on a _normal_ screen (i.e. non-HiDPI) these icons feel slightly "blurrier" than the old ones when using the viewer.

@rtestard ^^

**Comment by Snuffleupagus** [comment]
@calixteman Actually, if you move *only* the three new editing-icons into a separate PR I think we can just land those right away (since they are still an improvement over the placeholder icons).

**Comment by rtestard** [comment]
@Snuffleupagus Can you please share your screen resolution and dpi setting (i.e zoom level at 100%?)
Also does the bluriness appear only with the pdf.js icons or also with the Firefox icons on your screen?

**Comment by Snuffleupagus** [comment]
> Can you please share your screen resolution and dpi setting (i.e zoom level at 100%?)

My resolution is 1680x1050, however I'm not sure about the DPI value unfortunately (it's an Eizo S2202W monitor).
I know that my monitor is older, and using a non-HiDPI monitor is probably not very common these days. (I've stuck with it since most monitors I've tried are just way too bright for my eyes.)

> Also does the bluriness appear only with the pdf.js icons or also with the Firefox icons on your screen?

I have the same problem with some of the icons in the main Firefox UI not looking "sharp", and my setup is probably atypical nowadays, so I guess that this isn't entirely relevant here...

---

*Please note, just to make it absolutely clear:* The only reason that I didn't simply approve this PR is because it only updates a small portion of the toolbar/secondaryToolbar icons. Unless we actually update *all* icons, things will looks really inconsistent given the different style of the old/new icons.

**Comment by calixteman** [comment]
All the visible toolbar icons have been updated excepted "presentation mode" and "current view" ones (and the magnifying glass for the non-firefox viewer).
UI/UX will work on the secondary toolbar "later".

File: web/images/toolbarButton-openFile.svg
Comment (Snuffleupagus) [inline web/images/toolbarButton-openFile.svg:1]: This file contains lots of unnecessary stuff, that bloats the file-size, and which should be removed.

**Comment by timvandermeij** [comment]
I agree with that we either need to replace all icons or none of them. Mixing icon styles makes the UI feel inconsistent to me as well.

**Comment by rtestard** [comment]
> I agree with that we either need to replace all icons or none of them. Mixing icon styles makes the UI feel inconsistent to me as well. In the meantime [#15384 (comment)](https://github.com/mozilla/pdf.js/pull/15384#issuecomment-1235335739) sounds like a good idea to me to get the new icons in at least.

The issue is inconsistency in the UI. Ideally we would have refreshed all icons but given target to land on 106 we aligned with UX that having a consistent primary UI (visible toolbar elements) is more valuable than having a mix of new (annotation icons) and old (current icons) § this assumes users rarely open the overflow menu. We want all icons done eventually but given timelines we'll have to go with a compromise before we can land all refreshed icons.

**Comment by rtestard** [comment]
https://github.com/mozilla/pdf.js/issues/15401 will track implementation of icons from toolbar overflow and sidebar.

**Comment by Snuffleupagus** [comment]
> We want all icons done eventually but given timelines we'll have to go with a compromise before we can land all refreshed icons.

As asked previously, what schedule/timeline are we talking about here? When are *all* icons expected to updated?

(In this PR I'm being asked to approve changes that I personally disagree with, w.r.t. the partial update, so more clarity into the process/planning would definitely help here.)

**Comment by rtestard** [comment]
> > We want all icons done eventually but given timelines we'll have to go with a compromise before we can land all refreshed icons.
> 
> As asked previously, what schedule/timeline are we talking about here? When are _all_ icons expected to updated?
> 
> (In this PR I'm being asked to approve changes that I personally disagree with, w.r.t. the partial update, so more clarity into the process/planning would definitely help here.)

I just raised the UX request to get the icon set specced out. It took about 2 weeks last time so hopefully we have a fast turn§around on this and it can make 107. To be clear I agree with you with this not being ideal to only partially update icons, unfortunately this was a compromise we had to make in order to ship with 106.

File: web/images/toolbarButton-openFile.svg
Comment (Snuffleupagus) [inline web/images/toolbarButton-openFile.svg:2]: This doesn't look entirely correct, compared to all the other files, and probably explains why this one is "larger" in the GitHub preview; don't you want the following here as well?
```suggestion
<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment (Snuffleupagus) [reply web/images/toolbarButton-openFile.svg]: It doesn't seem that the update fully addressed this, as suggested, since the "header" of this file still contains a bunch of unnecessary stuff when compared to e.g. `web/images/toolbarButton-download.svg`.

---

Also, the updated `toolbarButton-menuArrow.svg` file is missing from the patch (it was present originally).

Comment (calixteman) [reply web/images/toolbarButton-openFile.svg]: Initially I got a 12x12 icon but I wrongly replaced it with the 16x16 one used for page-down.
I pinged the UX designer in charge of that stuff about that, but it was an holiday yesterday in US, so we should have one in today or tomorrow.

File: web/images/toolbarButton-openFile.svg
Comment (Snuffleupagus) [inline web/images/toolbarButton-openFile.svg:2]: As mentioned previously, can we please just change this as follows to be consistent with all of the other files?

```suggestion
<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

**Review by Snuffleupagus — APPROVED** [review]
r=me, with the comment addressed and assuming that you've tested that this works correctly in both light/dark themes.

File: web/images/toolbarButton-openFile.svg
Comment (calixteman) [reply web/images/toolbarButton-openFile.svg]: I got an icon from UX to replace the one I created.

**Pull request merged**
